poppy/provider/akamai/background_jobs/check_cert_status_and_update/check_cert_status_and_update_tasks.py

```diff
@@ -154,7 +238,16 @@ def execute(self, cert_obj_json):
 
                 if change_url not in pending_changes:
                     if cert_obj.domain_name in dns_names:
-                        return "deployed"
+                        if cert_obj.domain_name in latest_version_hosts:
+                            return "deployed"
```

Comment:
Please consider using a state variable and return at the end rather returning string

Reply:
Yes. I do wish to do the same for strings across the code. But, currently the code in this file seems to have the in-place strings for the statuses, error messages and log messages. Would be good to move all such string literals to a code related properties file so that we can have single source of truth for such fields (Later the same properties file can be used in test cases to access those values instead of declaring them again).